build-logic/publishing/src/main/kotlin/buildlogic.publish-to-central.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import buildlogic.cleanupMavenPom
+import org.gradle.api.publish.internal.PublicationInternal
+
+plugins {
+    id("java-library")
+    id("maven-publish")
+}
+
+val repoUrl = "https://github.com/sigstore/sigstore-java"
+
+publishing {
+    publications.withType<MavenPublication>().configureEach {
+        // Use the resolved versions in pom.xml
+        // Gradle might have different resolution rules, so we set the versions
+        // that were used in Gradle build/test.
+        versionMapping {
+            usage(Usage.JAVA_RUNTIME) {
+                fromResolutionResult()
+            }
+            usage(Usage.JAVA_API) {
+                fromResolutionOf("runtimeClasspath")
+            }
+        }
+        pom {
+            cleanupMavenPom()
+            name.set(
+                (project.findProperty("artifact.name") as? String)
```

Comment:
does this rename the pom? is like 73 in this file no longer necessary?

Reply:
What do you mean by "rename pom"?
Frankly speaking, I want to delete task `createReleaseBundle` completely, however, I would like to discuss that with you first, so I just kept the task just in case